pom.xml

```diff
@@ -51,6 +51,7 @@
     <htrace-core.version>3.1.0-incubating</htrace-core.version>
     <zookeeper.version>3.4.6.2.6.0.3-8</zookeeper.version>
     <dependency.maven-bundle-plugin.version>3.2.0</dependency.maven-bundle-plugin.version>
+    <karaf-maven-plugin.version>3.0.3</karaf-maven-plugin.version>
```

Comment:
Do not define a versio here. Use the karaf maven plugin from the plugin management in the parent pom.